engine/src/main/java/net/jqwik/engine/providers/EmailArbitraryProvider.java

```diff
@@ -15,7 +15,13 @@ public boolean canProvideFor(TypeUsage targetType) {
 
 	@Override
 	public Set<Arbitrary<?>> provideFor(TypeUsage targetType, ArbitraryProvider.SubtypeProvider subtypeProvider) {
-		return Collections.singleton(Arbitraries.emails());
+		Email email = targetType.findAnnotation(Email.class).get();
```

Comment:
Is this okay to get the email annotation? It works in the testcases.

Reply:
Thats correct this way.